### PR TITLE
Restore missing i18n values

### PR DESCRIPTION
**Pull request by aaronjudd**
This is a merge from master, and will also restore some keys that have
moved to new locations. However I’ve tried to vet and correct as much
as possible

### CODE REVIEW

File: private/data/i18n/en.json [hunk 1]
```diff
@@ -66,57 +180,83 @@
...
         "addVariantOptions": "Add options to enable 'Add to Cart' button",
         "inventoryQuantity": "Quantity",
-        "title": "Title",
+        "title": "Label",
         "price": "Price",
-        "optionTitle": "Option title",
+        "optionTitle": "Option",
         "barcode": "Barcode",
-        "compareAtPrice": "Compare at price",
+        "compareAtPrice": "MSRP",
```
Comment (spencern) [inline private/data/i18n/en.json]: Did we want to leave this as Compare At Price?

File: private/data/i18n/en.json [hunk 2]
```diff
@@ -66,57 +180,83 @@
...
         "productIsVisible": "Product is visible",
-        "productIsNotVisible": "Product is not visible",
-        "variantOptions": "Variant options"
+        "productIsNotVisible": "Product is not visible"
       },
       "productVariant": {
         "addVariantOptions": "Add options to enable 'Add to Cart' button",
         "inventoryQuantity": "Quantity",
-        "title": "Title",
+        "title": "Label",
```
Comment (spencern) [inline private/data/i18n/en.json]: I remember this change, I think we had decided to keep "Title" through out the variants and options.

File: private/data/i18n/en.json [hunk 3]
```diff
@@ -150,13 +300,10 @@
         "coreOrderWorkflow/created": "Created",
         "coreOrderWorkflow/processing": "Processing",
         "coreOrderWorkflow/completed": "Completed",
-        "coreOrderWorkflow/canceled": "Canceled",
-        "coreOrderWorkflow/picked": "Picked",
-        "coreOrderWorkflow/packed": "Packed",
```
Comment (spencern) [inline private/data/i18n/en.json]: are these duplicated elsewhere?

File: private/data/i18n/en.json [hunk 4]
```diff
@@ -174,62 +321,84 @@
...
           "new": "New",
-          "captured": "Captured",
+          "captured": "Payment captured",
+          "processing": "Processing",
           "shipped": "Shipped",
           "completed": "Completed",
           "canceled": "Canceled",
           "refunded": "Refunded"
         },
-        "capture": "Capture",
```
Comment (spencern) [inline private/data/i18n/en.json]: I think we still need most of these

File: private/data/i18n/en.json [hunk 5]
```diff
@@ -241,22 +410,93 @@
         "createOrLogin": "Create an account, or sign in",
         "guestOrLogin": "Sign in, or checkout as a guest",
         "continueAsGuest": "Continue as guest",
-        "guestMessage": "You can checkout without creating an account. You'll have a chance to create an account later.",
-        "addEmail": "Add email"
```
Comment (spencern) [inline private/data/i18n/en.json]: I think we still need this email message

File: private/data/i18n/en.json [hunk 6]
```diff
@@ -290,19 +539,59 @@
...
+          "billingCountry": "Country",
+          "billingPhone": "Phone",
+          "billingStatus": "Status",
+          "userEmails": "Email",
+          "orderDate": "Date",
+          "orderTotal": "Total"
+        }
+      },
       "accountsUI": {
-        "yourOrders": "Your orders",
```
Comment (spencern) [inline private/data/i18n/en.json]: These were added recently

**Comment by aaronjudd** [comment]
This is collaborative PR -> if you see something you know has changed. Fix it.  If you see something wrong. Fix it.  See something, do something.

File: private/data/i18n/en.json (hunk 6 above)
Comment (aaronjudd) [reply private/data/i18n/en.json]: This should have merged master into release-1.50, shouldn't have deleted anything new, just merged some old stuff back in, and potentially overwrote updates text with old text (mostly sentence case which I went through and fixed)

File: private/data/i18n/en.json (hunk 5 above)
Comment (aaronjudd) [reply private/data/i18n/en.json]: i18nKey="checkoutLogin.guestMessage"

not removed? ```      "checkoutLogin": {
        "welcome": "Welcome, it's so good to see you.",
        "loginCompleted": "Account completed",
        "createOrLogin": "Create an account, or sign in",
        "guestOrLogin": "Sign in, or checkout as a guest",
        "continueAsGuest": "Continue as guest",
        "guestMessage": "You can checkout without creating an account. You'll have a chance to create an account later."
      },```

File: private/data/i18n/en.json (hunk 1 above)
Comment (aaronjudd) [reply private/data/i18n/en.json]: There is actually more than one of these... I'm not sure which is correct.

File: private/data/i18n/en.json (hunk 4 above)
Comment (aaronjudd) [reply private/data/i18n/en.json]: Fixing.

File: private/data/i18n/en.json (hunk 5 above)
Comment (aaronjudd) [reply private/data/i18n/en.json]: (or did you mean addEmail).. Fixing.

**Pull request merged**